FINS/FINS-IOC-01App/Db/sans2d-vac-intervals.template

```diff
@@ -0,0 +1,79 @@
+## Distances (intervals) between baffles and detectors
+
+record(calc, "$(FULL_PREFIX)FDFB:INTERVAL"){
+    field(DESC, "Dist between front det and front baf")
+    field(SCAN, ".1 second")
+    field(EGU, "mm")
+    field(INPA, "$(P)MOT:FRONTDETZ")
+    field(INPB, "$(P)MOT:FRONTBAFFLEZ")
+    field(CALC, "b-a")
+    field(LOLO, "$(INTERVAL_MAJOR_WARNING)")
+    field(LLSV, "MAJOR")
+    field(LOW, "$(INTERVAL_MINOR_WARNING)")
+    field(LSV, "MINOR")
+}
+
+record(calc, "$(FULL_PREFIX)FDFBSP:INTERVAL"){
+    field(DESC, "Dist between front det and front baf sp")
+    field(SCAN, ".1 second")
+    field(EGU, "mm")
+    field(INPA, "$(P)MOT:FRONTDETZ:SP")
+    field(INPB, "$(P)MOT:FRONTBAFFLEZ:SP")
+    field(CALC, "b-a")
+    field(LOLO, "$(INTERVAL_MAJOR_WARNING)")
+    field(LLSV, "MAJOR")
+    field(LOW, "$(INTERVAL_MINOR_WARNING)")
+    field(LSV, "MINOR")
+}
+
+record(calc, "$(FULL_PREFIX)FBRB:INTERVAL"){
+    field(DESC, "Dist between front baf and rear baf")
+    field(SCAN, ".1 second")
+    field(EGU, "mm")
+    field(INPA, "$(P)MOT:FRONTBAFFLEZ")
+    field(INPB, "$(P)MOT:REARBAFFLEZ")
+    field(CALC, "b-a")
+    field(LOLO, "$(INTERVAL_MAJOR_WARNING)")
+    field(LLSV, "MAJOR")
+    field(LOW, "$(INTERVAL_MINOR_WARNING)")
+    field(LSV, "MINOR")
+}
+
+record(calc, "$(FULL_PREFIX)FBRBSP:INTERVAL"){
+    field(DESC, "Dist between front baf and rear baf sp")
+    field(SCAN, ".1 second")
+    field(EGU, "mm")
+    field(INPA, "$(P)MOT:FRONTBAFFLEZ:SP")
+    field(INPB, "$(P)MOT:REARBAFFLEZ:SP")
+    field(CALC, "b-a")
+    field(LOLO, "$(INTERVAL_MAJOR_WARNING)")
+    field(LLSV, "MAJOR")
+    field(LOW, "$(INTERVAL_MINOR_WARNING)")
+    field(LSV, "MINOR")
+}
+
+record(calc, "$(FULL_PREFIX)RBRD:INTERVAL"){
+    field(DESC, "Dist between rear baf and rear det")
+    field(SCAN, ".1 second")
+    field(EGU, "mm")
+    field(INPA, "$(P)MOT:REARBAFFLEZ")
+    field(INPB, "$(P)MOT:REARDETZ")
+    field(CALC, "b-a")
+    field(LOLO, "$(INTERVAL_MAJOR_WARNING)")
+    field(LLSV, "MAJOR")
+    field(LOW, "$(INTERVAL_MINOR_WARNING)")
+    field(LSV, "MINOR")
+}
+
+record(calc, "$(FULL_PREFIX)RBRDSP:INTERVAL"){
+    field(DESC, "Dist between rear baf and rear det sp")
+    field(SCAN, ".1 second")
+    field(EGU, "mm")
+    field(INPA, "$(P)MOT:REARBAFFLEZ:SP")
+    field(INPB, "$(P)MOT:REARDETZ:SP")
+    field(CALC, "b-a")
+    field(LOLO, "$(INTERVAL_MAJOR_WARNING)")
+    field(LLSV, "MAJOR")
+    field(LOW, "$(INTERVAL_MINOR_WARNING)")
+    field(LSV, "MINOR")
+}
```

Comment:
This feels like it could be templated a bit more, the calc record is almost identical for each case, just the pv names are changing. Could the pv names not be macros passed from the substitution file? It might be possible to further simplify by making the names the same e.g. using FB rather than FRONTBAFFEL in the motor so it is the same convention as the calc record naming and so only a single macro could be used to construct both PV names.